src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -596,6 +600,21 @@ public static void onRegisterKeyMappings(Options options) {
         ModLoader.get().postEvent(new RegisterKeyMappingsEvent(options));
     }
 
+    public static void onRegisterPictureInPictureRenderers(List<PictureInPictureRenderer<?>> renderers,
+                                                           MultiBufferSource.BufferSource bufferSource,
+                                                           ImmutableMap.Builder<Class<? extends PictureInPictureRenderState>, PictureInPictureRenderer<?>> builder) {
+        var modRenderers = new ArrayList<PictureInPictureRenderer<?>>();
+        RegisterPictureInPictureRendererEvent.BUS.post(new RegisterPictureInPictureRendererEvent(modRenderers, bufferSource));
```

Comment:
All of this can be in the Event's register function.

Reply:
Done